samples/bluetooth/enocean/README.rst

```diff
@@ -0,0 +1,114 @@
+.. _enocean_sample:
+
+Bluetooth: EnOcean
+##################
+
+The Bluetooth EnOcean sample demonstrates the basic usage of the :ref:`bt_enocean_readme` library.
+
+Overview
+********
+
+The EnOcean sample sets up a basic Bluetooth observer for both EnOcean switches and sensors.
+
+.. _enocean_sample_overview:
+
+.. include:: ../../../include/bluetooth/enocean.rst
+   :start-after: enocean_observing_output_start
+   :end-before: enocean_observing_output_end
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+* At least one :ref:`supported EnOcean device <bt_enocean_devices>`.
+
+.. note::
+    The sample supports up to four devices at a time that work with one board.
+    Using devices to work with different boards (or more than one board) is not supported.
```

Comment:
Perhaps it's too late in the day, but I have trouble parsing this sentence. I don't think it's needed, the first sentence in the notes is sufficient.